tensorflow/core/kernels/data/dataset_utils.h

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 #define TENSORFLOW_CORE_KERNELS_DATA_DATASET_UTILS_H_
 
 #include "tensorflow/core/framework/dataset.h"
+#include "tensorflow/core/framework/iterator.pb.h"
```

Comment:
depending on the proto header is no longer allowed in the TensorFlow code base ... I will need to make sense changes to make it possible for your to submit this PR ... I will notify you when the changes are done at which point you will need to merge them into your PR

Reply:
@jsimsa Thanks for your help on this! Feel free to let me know if there is anything I can help.